aip/general/0122.md

```diff
@@ -153,7 +153,7 @@ are used in contexts where the owning API is clear (for example,
 
 However, sometimes it is necessary for services to refer to resources in an
 arbitrary API. In this situation, the service **should** use the _full resource
-name_, a schemeless URI with the owning API's service endpoint, followed by the
+name_, a schemeless URI with the owning API's (logical) service name, followed by the
```

Comment:
```suggestion
name_, a schemeless URI with the owning API's service name, followed by the
```

I'm thinking doubling-down on the term `service name` within AIPs. In that context, "logical" might just make things more confusing for someone. So maybe remove it?